Release 16.12

ranche opened the pull request.

Commit:
SIPX-440: REST API improvement

Send phone profiles via rest

-send profile for one phone given id or MAC (if specified id not found, we assume the MAC is provided)
phones/{id}/sendProfile: send profile but do not restart phone
phones/{id}/sendProfile/restart: send profile and restart phone

example given id:
[mirceac@mircea build]$ curl --insecure -X PUT https://superadmin:11111111@localhost/sipxconfig/api/phones/70/sendProfile
[mirceac@mircea build]$ curl --insecure -X PUT https://superadmin:11111111@localhost/sipxconfig/api/phones/70/sendProfile/restart

example given mac:
[mirceac@mircea build]$ curl --insecure -X PUT https://superadmin:11111111@localhost/sipxconfig/api/phones/0004f28125e4/sendProfile
[mirceac@mircea build]$ curl --insecure -X PUT https://superadmin:11111111@localhost/sipxconfig/api/phones/0004f28125e4/sendProfile/restart

-send profile for a list of phone ids
phones/sendProfile: send profile but do not restart phones
phones/sendProfile/restart: send profile and restart phones

example:
curl --insecure -X PUT -H "Content-Type: application/json" -d '{"ids":[66,67,68,69,70,71,72,73]}' https://superadmin:11111111@localhost/sipxconfig/api/phones/sendProfile
curl --insecure -X PUT -H "Content-Type: application/json" -d '{"ids":[66,67,68,69,70,71,72,73]}' https://superadmin:11111111@localhost/sipxconfig/api/phones/sendProfile/restart

Commit:
SIPX-440: forgot to add a class

Commit:
UC-3837: 15.10 invalid /etc/hosts entry

- generate entries from config and insert files using cfengine

Commit:
UC-3837: 15.10 invalid /etc/hosts entry

- generate entries from config and insert files using cfengine

Commit:
SIPX-438: REST API bug

-fix phone line rest api

avilcu committed:
UC-3904 Improve voicemail cleanup procedure

Conflicts:
	sipXsupervisor/etc/logrotate.cf

avilcu committed:
UC-3904 Improve voicemail cleanup procedure

Commit:
UC-3939: investigate why IVR needs to be at DEBUG verbosity to remain stable

- remove export_vars call, export needed vars inline in bridge command

Commit:
UC-3939: investigate why IVR needs to be at DEBUG verbosity to remain stable

- remove export_vars call, export needed vars inline in bridge command

Commit:
SIPX-465: Sipxivr stopped after update from 16.02 to 16.04

- write service cfengine class

Commit:
SIPX-330: Managing users in phonebooks

- add button to empty phonebook - patch from João Veríssimo, thanks!

Commit:
SIPX-182: Multiple prefixes are not working under dialplan

-remove old help text

Commit:
SIPX-214: REST: MoH

-add rest api to detect user permission to set moh setting
case 1: user not part of any group, user moh permission true:
user can change moh setting
case 2: user part of at least one group, user can change moh setting only if
user group moh permission is true

avilcu committed:
UC-3413 Add Reach recipe criteria for tick interval = 0
add validation bean for digits only no 0

Commit:
UC-3866: zen 5201: snmptrapd.conf is missing the webui snmp community string change

-remove community string column from alarm_receiver table, use the community string setting instead

mharcar committed:
unitelite 16.04 minimized

avilcu committed:
Bump 16.08

Commit:
UC-3968: Proxy should add "transport=tcp" into Record-Route of responses (180, 200) in case TCP usage

- expose bridge-proxy transport setting

Commit:
UC-3968: Proxy should add "transport=tcp" into Record-Route of responses (180, 200) in case TCP usage

- expose bridge-proxy transport setting

Commit:
SIPX-3: Double registration entries and incoming calls intermittently going to VM while on TLS and cluster

replicate all authorities files on all nodes in HA env

mharcar committed:
UW-336 unite lite 16.04

Commit:
Bump 16.08

Commit:
Bump 16.08

Commit:
UC-3899: In System -> Backup be able to Enable/Disable Recordings Backup

-fix issue when after restore data remains in tmp directory by clear tmp restore content after restore process

Commit:
Merge branch 'release-16.04' into release-16.08

avilcu committed:
UC-3904 Improve voicemail cleanup procedure
voicemail_cleanup parameter should be displayed in advanced view

avilcu committed:
UC-3904 Improve voicemail cleanup procedure
voicemail_cleanup parameter should be displayed in advanced view

Commit:
SIPX-330: Managing users in phonebooks

- added missing labels

avilcu committed:
Merge remote-tracking branch 'origin/release-15.12' into release-16.02

avilcu committed:
Merge remote-tracking branch 'origin/release-16.02' into release-16.04

avilcu committed:
UC-1726 Increase max value for voice mail imports
increased to 10GB

avilcu committed:
Merge remote-tracking branch 'origin/release-15.12' into release-16.02

avilcu committed:
Merge remote-tracking branch 'origin/release-16.02' into release-16.04

Commit:
Add Upstream repo as docker build param

Commit:
Add Upstream repo as docker build param

Conflicts:

	build_docker_rpm.sh

Commit:
Add Upstream repo as docker build param

Conflicts:

	build_docker_rpm.sh

mharcar committed:
UW-133 unite lite 16.04

Commit:
SIPX-474: voicemails in safari are played as live broadcast

-added requested headers: Content-Range and Accept-Ranges

Commit:
Merge branch 'release-15.12' into release-16.02

avilcu committed:
UC-3880 Move Network Packet Capture Service to Diagnostic Tab

avilcu committed:
UC-3715 Show Reach Agent record in search results for the admin UI

avilcu committed:
SIPX-404 P-Preferred-Identity of Proxy - gui work

avilcu committed:
SIPX-399 Agent state not available for duplicated agents

avilcu committed:
UC-3413 Add Reach recipe criteria for tick interval = 0

bmachens committed:
SIPX-477 Add "voIpProt.SIP.CID.sourcePreference" to Polycom Plugin (FW 3.x, FW 4.x, FW 5.x)

bmachens committed:
SIPX-477 Add description from Polycom UC Admin Guide to parameter

Commit:
Merge pull request #53 from bmachens/release-16.04

SIPX-477 Polycom - Display Caller ID

Commit:
Portuguese localization

Commit:
bump version

avilcu committed:
Merge remote-tracking branch 'origin/release-16.04' into release-16.08

yozmag committed:
Add Portuguese Region

avilcu committed:
SIPX-401 DID in FAX, Hunt Groups Voicemail, etc., are not indexed

Commit:
Merge pull request #54 from yozmag/patch-1

SIPX-478: Add Portuguese Region

Commit:
SIPX-478 - portuguese lang pack

Commit:
SIPX-470: Live Attendant not updated in Mongo properly

- fix contact replication

Commit:
SIPX-470: Live Attendant not updated in Mongo properly

- fix contact replication

Commit:
SIPX-470: Live Attendant not updated in Mongo properly

- fix contact replication

Commit:
Merge remote-tracking branch 'origin/release-16.04' into release-16.08

Commit:
Bump version

avilcu committed:
SIPX-481 Add support for Polycom 301, 311, 401, 411, 501, 601

avilcu committed:
UC-3901 Restore operation of 15.10 backup configuration to 15.12 fails
do not fail restore if elasticsearch archive not found in archive

avilcu committed:
Merge remote-tracking branch 'origin/release-15.12' into release-16.02

avilcu committed:
Merge remote-tracking branch 'origin/release-16.02' into release-16.04

avilcu committed:
Merge remote-tracking branch 'origin/release-16.04' into release-16.08

avilcu committed:
SIPX-482 Can't remove Phone group from Intercom,Groups field can't be

Commit:
UC-4025: SOAP api bug - duplicate entries

-fixed add phone line
-fixed phone delete line in SOAP service (update remaining lines position after deletion)

Commit:
Merge branch 'release-15.12' into release-16.04

Commit:
Merge branch 'release-16.04' into release-16.08

Commit:
UC-4025: SOAP api bug - duplicate entries

-fix style

Commit:
Merge branch 'release-15.12' into release-16.04

Commit:
Merge branch 'release-16.04' into release-16.08

Commit:
SIPX-478: update submodule

avilcu committed:
UC-4019 Call park name extension creates a duplicate entry in mongo db

Commit:
UC-4002 Proxy should add "transport=tcp" into Record-Route
- incoming transport check replaced with outgoing transport check

avilcu committed:
UC-3903 zen 5294 : improvement for default line labels in Polycom 5.x

avilcu committed:
UC-3870 zen 5222 : CDR csv export repeats page headers within rows

avilcu committed:
Fixed style for precommit

avilcu committed:
UC-4030 If a user puts call queue extension in speed dial and subscribes
to presence this SSS has problems

Commit:
SIPX-484: slow queries due to instrument field not indexed in registration db

- ensure index on instrument field

Commit:
SIPX-484: slow queries due to instrument field not indexed in registration db

- ensure index on instrument field

Commit:
SIPX-474: voicemails in safari are played as live broadcast

-add standard header Accept-Ranges: bytes
(restlet is protocol agnostic so adding standard headers is done by the framework)

mharcar committed:
UW-334 UW-310 UW-331 UW-328 UW-338 16.08 unitelite version

Commit:
SIPX-228: provision jitsi

- install jitsi RPM automatically

Commit:
SIPX-493: Configuration Generation Error after upgrade from 16.02 to 16.04

Commit:
Revert "SIPX-493: Configuration Generation Error after upgrade from 16.02 to 16.04"

This reverts commit 79dee453dc8f30a2ec9d9e9acf53571aaa35c1c8.

Commit:
SIPX-493: Configuration Generation Error after upgrade from 16.02 to 16.04

Commit:
SIPX-493: Configuration Generation Error after upgrade from 16.02 to 16.04

yozmag committed:
Adding new ring types

Adding new Ring types according to yealink configuration parameters V80

yozmag committed:
Adding new ring types

Adding new Ring types according to yealink configuration parameters V80

yozmag committed:
Adding new ring types

Adding new Ring types according to yealink configuration parameters V8
Example:
account.1.ringtone.ring_type = Ring3.wav
It means configuring Ring3.wav for account1.
account.1.ringtone.ring_type = Common
It means account1 will use the ring tone selected for the IP phone configured by the parameter "phone_setting.ring_type".

yozmag committed:
Adding new ring types

Adding new Ring types according to yealink configuration parameters V8
Example:
account.1.ringtone.ring_type = Ring3.wav
It means configuring Ring3.wav for account1.
account.1.ringtone.ring_type = Common
It means account1 will use the ring tone selected for the IP phone configured by the parameter "phone_setting.ring_type".

Commit:
SIPX-494 Adding new ring types

SIPX-494 Adding new ring types

Commit:
SIPX-494 Adding new ring types

SIPX-494 Adding new ring types

mharcar committed:
UW-338 UW-340 and full unite lite version

Commit:
UC-4055: consider tcp keepalive reduction

- see https://docs.mongodb.com/v2.6/faq/diagnostics/#does-tcp-keepalive-time-affect-sharded-clusters-and-replica-sets

Commit:
SIPX-478: update submodule

Commit:
Merge remote-tracking branch 'origin/release-16.04' into release-16.08

Commit:
SIPX-501: With a valid/signed certificate,connecting over HTTPS will prompt you to accept an invalid cert localhost.locadomain

-fix zz_apache.cf to replicate ssl.conf and ssl-web.crt in apache. (read from cfdata location path and not from cfdata global path)

Commit:
SIPX-496: allow MP3 bitrate setting in IVR UI

- new settings tab in Media Services page with sample rate, bit rate and recording quality settings

Commit:
UC-1743: Investigate DST updates in schedules, voicemails, cdrs, etc.

-ensure fixed rate and an initial delay of 10 seconds for the DST timer
-substract minutes above the estimated DST time value (estimation is allways after DST)

Commit:
UC-4116: Hack to restart SSS no matter if SSS config is changed at every server profile sending

-create new config generation notify/subscribe event. mark send profiles true/false

Commit:
UC-4116: Hack to restart SSS no matter if SSS config is changed at every server profile sending

-create new config generation notify/subscribe event. mark send profiles true/false

Conflicts:
	sipXconfig/neoconf/src/org/sipfoundry/sipxconfig/ivr/IvrConfig.java
	sipXconfig/neoconf/src/org/sipfoundry/sipxconfig/speeddial/SpeedDialManagerImpl.java

Commit:
UC-4116: Hack to restart SSS no matter if SSS config is changed at every server profile sending

-remove send profile dependency from ConfigManager worker

Commit:
UC-4116: Hack to restart SSS no matter if SSS config is changed at every server profile sending

-remove send profile dependency from ConfigManager worker

mharcar committed:
UW-334 UW-328 UW-342 UW-310 unitelite 16.08 release

Commit:
UC-3847 configuration of surefire build added to pom.xml

Commit:
SIPX-496: recording settings

- change sample rate default value to 16000

Commit:
Revert "SIPX-496: recording settings"

This reverts commit adec4ffdd8f7d3f6b355c7f925541802c72085d7.

Commit:
SIPX-496: allow MP3 bitrate setting in IVR UI

- change conference default rate to 16000

Commit:
UC-3847 selenium and surefire versions updated

Commit:
UC-4104: mongo c++ driver fix for tcp connection issue

Commit:
UC-4104: mongo c++ driver fix for tcp connection issue

Commit:
UC-4104: mongo c++ driver fix for tcp connection issue

Commit:
Merge branch 'release-16.08' of https://github.com/sipXcom/sipxecs into release-16.08

Commit:
UC-4030: If a user puts call queue extension in speed dial and subscribes to presence this SSS has problems

Commit:
SIPX-504:  Missing CallerId in audiocodes

Commit:
SIPX-496: allow MP3 bitrate setting in IVR UI

- remove unsupported sample rate and bit rate values

Commit:
UC-1743: Investigate DST updates in schedules, voicemails, cdrs, etc.

-ensure voicemail dial-plan reconfig when DST changes

Commit:
UC-4114: Deleting Voicemails from email doesn't notify MWI

-mark message heard when deleting. also move it to delete folder instead of removing it from the system
-show in web page information about user action

Commit:
Revert "UC-4114: Deleting Voicemails from email doesn't notify MWI"

This reverts commit 26c59bda74628c0bf7997827633a88824d6f3e15.

Commit:
UC-4114: Deleting Voicemails from email doesn't notify MWI

-new rest action movetodeleted
-show deletion information if successful

Commit:
UC-4114: Deleting Voicemails from email doesn't notify MWI

-new rest action movetodeleted
-show deletion information if successful

Commit:
SIPX-496: allow MP3 bitrate setting in IVR UI

- use proper values (kb/s) for encode-brate setting

Commit:
SIPX-485: Move some Polycom Configuration Items to Advanced

mharcar committed:
UW-343 UW-344 unitelite 16.08

Commit:
UW-320: Referential URL for Logout

-change logindetailsResourceWithPin rest api to use configured address when logging out
and stick with config login page if empty

Commit:
UW-320: Referential URL for Logout

-use apache commons

Commit:
UC-4030:  If a user puts call queue extension in speed dial and subscribes to presence this SSS has problems

A user should be able to subscribe to presence for a valid system extension, a valid system park orbit and a valid SIP address.
Not allowed: Call Queue extensions, Phone numbers not on system

Commit:
SIPX-496: allow MP3 bitrate setting in IVR UI

cristianmoisa committed:
UC-3847

cristianmoisa committed:
UC-3847 - added ID for Backup->Local Backup->Files Voicemail/Configuration/CDR

cristianmoisa committed:
UC-3847 - added ID for Backup->Local Backup->Files Voicemail/Configuration/CDR

mharcar committed:
UW-344 unitelite 16.08

cristianmoisa committed:
UC-3847 - Backup Local Configuration Test

cristianmoisa committed:
Merge remote-tracking branch 'origin/release-16.08' into release-16.08

mharcar committed:
UW-310 unitelite 16.08 release

Commit:
SIPX-399 Agent state not available for duplicated agents

mharcar committed:
UW-345 unitelite 16.08 release

Commit:
BUmp 16.12

Commit:
UC-4156: reload FS only (instead restarting) when local stream config file modified

Commit:
UC-4156: reload FS only (instead restarting) when local stream config file modified

Commit:
Bump 16.12

Commit:
UC-4001: Strip leading characters from username in LDAP import

Commit:
SIPX-507: Callqueue doesnt show queue's name as callerid

- add freeswitch-lua as RPM dep

Commit:
SIPX-507: Callqueue doesnt show queue's name as callerid

- add freeswitch-lua as RPM dep

Commit:
UC-4158: REST API improvement 2

-fix setNotified
-fix do not update null properties
-introduced new class to update not null properties

Commit:
Merge branch 'release-15.12' into release-16.04

Commit:
Merge branch 'release-15.12' into release-16.08

Commit:
Merge branch 'release-16.08' into release-16.12

Commit:
SIPX-389: Cannot acess HTML reports

-instruct httpd not to double encode URL / character

Commit:
Merge branch 'release-16.04' into release-16.08

Commit:
Merge branch 'release-16.08' into release-16.12

bmachens committed:
SIPX-509 Removed hard coded firmwares to load correct default value for FW 4.x and 5x
Removed hard coded FW for VVX 300, 310, 400, 410 and 600 to get the correct default firmware

Commit:
Merge pull request #61 from bmachens/release-16.08

SIPX-509

Commit:
UC-4160: zen 5966 : webui with IM doesn't display callerid number in voicemail page

-add callerid name and callerid number when depositing a voicemail

Commit:
UC-4193: zen 6013 : StateQueueClient 100ms hard coded timeout

-added tcp-timeout setting into etc/sipxpbx/sipxsqa-config
default is 100 ms

Commit:
UC-4193 StateQueueClient 100ms hard coded timeout
- refactoring before actual fix:
- BlockingTcpClient moved to separate source/include files
- friendship of StateQueueClient and BlockingTcpClient removed to avoid interface confusion
- some of the BlockingTcpClient methods made private

Commit:
UC-4193: zen 6013 : StateQueueClient 100ms hard coded timeout

-replicate tcp timeout in sipxsqa-client.ini too

Commit:
UC-4193 StateQueueClient 100ms hard coded timeout
- magic numbers removed
- clean unused code from BlockingClient
- read of "tcp-timeout" option added

Commit:
UC-4193 StateQueueClient 100ms hard coded timeout
- magic numbers removed
- clean unused code from BlockingClient
- read of "tcp-timeout" option added

Commit:
UC-4193 StateQueueClient 100ms hard coded timeout
- magic numbers removed
- clean unused code from BlockingClient
- read of "tcp-timeout" option added

Commit:
Merge branch 'release-16.12' of https://github.com/sipXcom/sipxecs into release-16.12

Commit:
Merge branch 'release-16.12' of https://github.com/sipXcom/sipxecs into release-16.12

Commit:
UC-4193 StateQueueClient 100ms hard coded timeout
 - BlockingTcpClient.cpp merged back to BlockingTcpClient.h to avoid compiltaion issues

Commit:
SIPX-511: Certificate issue with openfire/IM after fresh install

Commit:
Merge branch 'release-16.08' into release-16.12

Commit:
UC-4208: Enabling or disabling live attendant from TUI

- enabling / disabling night mode should not affect live aa enable / disable

Commit:
UC-4208: Enabling or disabling live attendant from TUI

- automatically disable night mode when live aa enabled

mharcar committed:
UW-350 UW-347 unitelite 16.12

Commit:
UC-4213: Set default ring for to 20

cristianmoisa committed:
UC-3847 - Backup Test more edits

cristianmoisa committed:
Merge branch 'release-16.12' of https://github.com/sipXcom/sipxecs into release-16.12

cristianmoisa committed:
UC-3847 Backup test working now..

joegen commented:
The legacy code base is no longer active.  Though, since this merge is clean, I don't see any reason why it can't be accepted. Any particular reason why you want to merge down stream commits?

Commit:
Update README

Commit:
Update README

Commit:
UC-4189: barge in functionality broken

mharcar committed:
UW-338 UC-4216 UW-371 UW-231 unitelite 16.12

mharcar committed:
UW-338 UC-4216 UW-371 UW-231 unitelite 16.12 fixed previous commit

Commit:
SIPX-517: Mongo plugin not entirely ported to open source

Commit:
SIPX-517: Mongo plugin not entirely ported to open source

Commit:
SIPX-517: Mongo plugin not entirely ported to open source

Commit:
UC-4240: zen 6146: backups to run under cgroups

-memory limitation for backup and mongodump

mharcar committed:
UW-372 UW-378 UW-184 unitelite 16.12

mharcar committed:
UW-378 UC-4160 unitelite 16.12

mharcar committed:
UW-374 unitelite 16.12

Commit:
UC-4044 uniteme setup to bind network interface of choice
- preparation for IP moval script development.

mharcar committed:
UW-381 unitelite 16.12

senescu committed:
UC-3847 - Added User Group Create/Delete/Verify test

cristianmoisa committed:
UC-3847 - User logout test

cristianmoisa committed:
UC-3847 - Admin Role Create/Delete/Verify test

senescu committed:
UC-3847 - Added Add remove user from group test

cristianmoisa committed:
UC-3847 - Admin Role Assign/Remove test

cristianmoisa committed:
Merge remote-tracking branch 'origin/release-16.12' into release-16.12

# Conflicts:
#	sipXverify/src/org/sipxcom/verify/AbstractTest.java

bmachens committed:
SIPX-524 Yealink DND Local Processing

Commit:
Merge pull request #62 from bmachens/release-16.12

SIPX-524 Yealink DND Local Processing

senescu committed:
UC-3847 - Added User Group Create/Delete/Verify test + Missing File

senescu committed:
Merge remote-tracking branch 'origin/release-16.12' into release-16.12

cristianmoisa committed:
UC-3847 - Updated some tests

Commit:
UC-4240: zen 6146: backups to run under cgroups

-change help text to specify memory type limitation (RAM)

Commit:
SIPX-525: fix path to mod_com_g729 freeswitch module

Commit:
SIPX-523: Make meeting entry and exit tone optional
- add two new options in conf setup -  play on exit / enter

Commit:
UC-4267 - Config generation fails with external Line on Phone

cristianmoisa committed:
UC-3847 - Added support for Vibe app with Sikuli

cristianmoisa committed:
UC-3847 - Added support for Vibe app with Sikuli tool

cristianmoisa committed:
Merge remote-tracking branch 'origin/release-16.12' into release-16.12

Commit:
SIPX-521: prompt for user name on entry to conference

- add Admin UI options, add proper FS XML dialplan

Commit:
SIPX-521: prompt for user name on entry to conference

Commit:
SIPX-522: Play name of person leaving conference
- added lua script to play and remove file
- unit tests

Commit:
IPX-522: Play name of person leaving conference
- fix lua script

Commit:
Revert "IPX-522: Play name of person leaving conference"

This reverts commit a51f68ec7627a6fcbb08adee90d174257c05ae78.

Commit:
SIPX-522: Play name of person leaving conference

cristianmoisa committed:
UC-3847 - Sikuli stuff

cristianmoisa committed:
Merge remote-tracking branch 'origin/release-16.12' into release-16.12

cristianmoisa committed:
UC-3847 - Updated vibe screenshots

cristianmoisa committed:
UC-3847 - Updated some tests

cristianmoisa committed:
UC-3847 - Updated some tests

cristianmoisa committed:
UC-3847 - Vibe Login as registered user, logout, login as guest user, different logout.

cristianmoisa committed:
UC-3847 - Deleted/Moved VIBE stuff to other repo

bmachens committed:
SIPX-529 Allow to self should only override name check for BLF, not BLF/Speed dial check

bmachens committed:
Merge branch 'release-16.12' of https://github.com/bmachens/sipxecs into release-16.12

dizzy committed:
Merge pull request #65 from bmachens/release-16.12

SIPX-529

bmachens committed:
SIPX-530 Add phone models of Polycom VVX 301, 311, 401, 411, 501 and 601 to Auto provision servlet

dizzy committed:
Merge pull request #66 from bmachens/release-16.12

SIPX-530

Commit:
SIPX-530 Add phone models of Polycom VVX 301, 311, 401, 411, 501 and 601 to Auto provision servlet

- fix compiler error

Commit:
SIPX-533: AA auto attendant not restricted by locations (again)

- generate mongo entires always (not only on live attendant), use expiration of 0 to direct to AA when live attendant not enabled

Commit:
SIPX-534: Under Dialing Rules->AA page , the selected locations get cleared out when enabling/disabling live attendant

Commit:
Fix ISO build

bmachens committed:
SIPX-539 Yealink Emergency DND Feature

bmachens committed:
SIPX-539 Yealink DND Emergency Localization

bmachens committed:
SIPX-593 Fix last Localization commit

dizzy committed:
Merge pull request #69 from iant-gmbh/release-16.12

SIPX-539

bmachens committed:
SIPX-540 Call Number Filter Provisioning for Yealink

dizzy committed:
Merge pull request #70 from iant-gmbh/release-16.12

SIPX-540 Call Number Filter Provisioning for Yealink

bmachens committed:
Search Request Uri Tag "x-sipx-alert-info" to set Alert-Info Header for external Calls

bmachens committed:
Use From URI instead of Request URI

dizzy committed:
Merge pull request #71 from iant-gmbh/release-16.12

SIPX-560

Commit:
UC-4307: Add new settings for User Portal configuration

-add user portal settings

Commit:
UC-3683: REST API to manage user and group settings

Commit:
UC-4328: REST API to manage user properties for user portal

-new rest api for user: api/my/user only GET is allowed

Commit:
UC-4313: Conference settings REST API to include new parameters

add following settings to be changed via REST: sipxconfig/rest/my/conferences/<conf_name>
<setting>
    <playEntryToneEnabled>false</playEntryToneEnabled>
    <playExitToneEnabled>false</playExitToneEnabled>
    <recordAndPlayNameOnEntryEnabled>true</recordAndPlayNameOnEntryEnabled>
    <recordAndPlayNameOnExitEnabled>true</recordAndPlayNameOnExitEnabled>
</setting>

Commit:
UC-4307: remove Type from names

mharcar committed:
UW-355 UW-367 UW-379 UC-4290 UW-383 UC-4290 UW-384 UW-385 UW-387 UW-386 UW388 UC-4324 unitelite backport to 16.12 stage

mharcar committed:
UW-388 unitelite 16.12 stage

Commit:
UC-4376: Disable MWI subscription if Voicemail permissions are disabled

Commit:
Bumping release 16.12.1

Commit:
SIPX-617: Make phone line label configurable

add first/last name automatic generation on polycom phones firmware 5

Commit:
UC-4398: zen 6798: VVX 1500 profile voice codec typo

-change ksps into kbps

Commit:
UC-4472 Proxy Segfault caused by malformed request URI

Commit:
SIPX-479: always write aa_live in mappingrules xml file

Commit:
SIPX-479: fix unit tests
